src/components/Search/OmniBox.js

```diff
@@ -91,18 +91,18 @@ export class OmniBox extends Component {
       // to the search page when we redirect).
 
       push({ pathname: 'search', search: locationSearch });
+
```

Comment:
@bbfsdev @YuriHechter I don't see how it doesn't reset the query on every new search.
We don't want to reset the filters on the search results page. Only on home page